blog-service/2023-11-06-monitors.md

```diff
@@ -0,0 +1,15 @@
+---
+title: Time Zone Settings for Monitors (Alerts)
+image: https://www.sumologic.com/img/logo.svg
+keywords:
+  - monitors
+  - alerts
+hide_table_of_contents: true
+authors:
+  - url: https://help.sumologic.com/release-notes-service/rss.xml
+    image_url: /img/release-notes/rss-orange.png
+---
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+We've rolled out the ability to designate a **Preferred Time Zone** when you're setting up alert monitor email notifications. [Learn more](/docs/alerts/monitors/create-monitor#step-3-notifications-optional).
```

Comment:
This is not just for email notifications. It will apply to all notification types (webhook, pagerduty, etc)

Reply:
thanks for clarifying! i'll change this.